Jenkinsfile

```diff
@@ -2,6 +2,23 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
 node {
   stage ("Start") {
+    // Check if the user who opened the PR is a known collaborator (i.e., has been added to a hyrise/hyrise team)
+    try {
+      withCredentials([usernamePassword(credentialsId: '5fe8ede9-bbdb-4803-a307-6924d4b4d9b5', usernameVariable: 'GITHUB_USERNAME', passwordVariable: 'GITHUB_TOKEN')]) {
```

Comment:
Is `GITHUB_TOKEN` set in the Jenkins settings, e.g., via the web interface?

Reply:
No, it is set right here. We pass in the name of a variable as a string. A superior concept, which should definitely find its way into C++!